From: Alec Leamas <leamas.alec@gmail.com>
Date: Tue, 28 Aug 2018 08:50:01 +0200
Subject: Build: Streamline some diagnostics, print cmake version.

Origin: upstream https://github.com/OpenCPN/OpenCPN/commit/d4059ec067567
Applied-Upstream: 5.0.0

cmake version is needed when building on remote builders.
Change messages about "library not found" the more
meaningful "using bundled library...". Use the term bundled
consistently.
---
 CMakeLists.txt                      | 17 +++++++++--------
 plugins/chartdldr_pi/CMakeLists.txt | 16 ++++++++--------
 2 files changed, 17 insertions(+), 16 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7b990ab..7f4d13f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,6 +40,7 @@ ENDIF (COMMAND cmake_policy)
 PROJECT(OpenCPN)
 
 SET(CMAKE_VERBOSE_MAKEFILE ON)
+message(STATUS "cmake version: ${CMAKE_VERSION}")
 
 IF(APPLE)
   SET(PACKAGE_NAME OpenCPN)
@@ -674,8 +675,8 @@ IF(NOT QT_ANDROID)
         FIND_PACKAGE(CURL REQUIRED)
         INCLUDE_DIRECTORIES(${CURL_INCLUDE_DIR})
     ELSE(OCPN_USE_EXTERN_CURL)
-        INCLUDE("Curl.cmake")
-	MESSAGE (STATUS "Using local curl library...")
+      INCLUDE("Curl.cmake")
+      MESSAGE (STATUS "Using bundled curl library...")
     ENDIF(OCPN_USE_EXTERN_CURL)
     SET(SRC_WXCURL
         src/wxcurl/base.cpp
@@ -1313,13 +1314,13 @@ ENDIF(OPENGL_FOUND)
 
 
 IF(TINYXML_FOUND)
-    message (STATUS "Building with tinyxml includes")
-    INCLUDE_DIRECTORIES(${TINYXML_INCLUDE_DIR})
-    ADD_DEFINITIONS(-DTIXML_USE_STL)
+  message (STATUS "Building with system tinyxml")
+  INCLUDE_DIRECTORIES(${TINYXML_INCLUDE_DIR})
+  ADD_DEFINITIONS(-DTIXML_USE_STL)
 ELSE(TINYXML_FOUND)
-    message (STATUS "tinyxml library not found")
-    SET(SRCS ${SRCS} ${SRC_TINYXML})
-    ADD_DEFINITIONS(-DTIXML_USE_STL)
+  message (STATUS "Building with bundled tinyxml")
+  SET(SRCS ${SRCS} ${SRC_TINYXML})
+  ADD_DEFINITIONS(-DTIXML_USE_STL)
 ENDIF(TINYXML_FOUND)
 
 
diff --git a/plugins/chartdldr_pi/CMakeLists.txt b/plugins/chartdldr_pi/CMakeLists.txt
index 5ed46cf..56bcebc 100644
--- a/plugins/chartdldr_pi/CMakeLists.txt
+++ b/plugins/chartdldr_pi/CMakeLists.txt
@@ -143,14 +143,14 @@ IF(TINYXML_FOUND)
  ADD_LIBRARY(${PACKAGE_NAME} SHARED ${SRC_UNARR} ${SRC_CHARTDLDR})
  TARGET_LINK_LIBRARIES(${PACKAGE_NAME} ${TINYXML_LIBRARIES})
 ELSE(TINYXML_FOUND)
- message (STATUS "Chartdldr_pi building with embedded tinyxml")
- INCLUDE_DIRECTORIES(src/tinyxml/)
- SET(SRC_LTINYXML
-            src/tinyxml/tinyxml.cpp
-            src/tinyxml/tinyxmlparser.cpp
-            src/tinyxml/tinyxmlerror.cpp
- )
- ADD_LIBRARY(${PACKAGE_NAME} SHARED ${SRC_UNARR} ${SRC_LTINYXML} ${SRC_CHARTDLDR})
+  message (STATUS "Chartdldr_pi building with bundled tinyxml")
+  INCLUDE_DIRECTORIES(src/tinyxml/)
+  SET(SRC_LTINYXML
+             src/tinyxml/tinyxml.cpp
+             src/tinyxml/tinyxmlparser.cpp
+             src/tinyxml/tinyxmlerror.cpp
+  )
+  ADD_LIBRARY(${PACKAGE_NAME} SHARED ${SRC_UNARR} ${SRC_LTINYXML} ${SRC_CHARTDLDR})
 ENDIF(TINYXML_FOUND)
 ADD_DEFINITIONS(-DTIXML_USE_STL)
 
